app/cypress/integration/cif/project-revision/create-first-project-revision.spec.js

```diff
@@ -11,14 +11,15 @@ describe("when creating a project, the project page", () => {
     cy.get("button").contains("Add a Project").click();
   });
 
-  it("allows an admin user to create a project", () => {
+  it("allows an admin user to create an EP project", () => {
     //add new
     cy.fillAndCheckNewProjectForm("Emissions Performance", "2020");
-    cy.happoAndAxe("Project New Form", "filled", "main");
+    cy.happoAndAxe("EP Project New Form", "filled", "main");
     cy.findByRole("button", { name: /^confirm/i }).click();
 
     // add overview
-    cy.findByText("Emissions Performance").should("have.length", 1);
+    cy.url().should("include", "/form/0");
+    cy.findByText("Emissions Performance - 2020");
     cy.fillOverviewForm(
```

Comment:
The original should have been fine, but I was getting a lot of flakiness, so hopefully this helps long-term